### ISSUE TITLE
Feature. Add path to requirements.txt

### ISSUE THREAD

**Issue by Niccolum** [FHPythonUtils/LicenseCheck#9]
In my project I use next requirements hierarchy:
requirements/base.txt
requirements/dev.txt
requirements/docs.txt
requirements/test.txt
requirements/lint.txt

I want to check one or multiple files in this hierarchy, but according 2021.2 release - I can't.
Same hierarchy we can see, for example, here:
https://github.com/aio-libs/aiohttp/tree/master/requirements
can you add optional flag path to file?

**Comment by FredHappyface** [comment]
Yeah can do, I'll add that to my to-do list

**Comment by FredHappyface** [comment]
Do you only foresee needing to check one requirements.txt at a time or will you be wanting to pass in a list of these?

At the moment I am thinking of implementing the api like this:
```
licensecheck --using requirements:path/to/reqs.txt
```

My proposal for multiple requirements would be:
```
licensecheck --using requirements:path/to/reqs.txt;path/to/other/reqs.txt
```

The drawback would be that `:` and `;` become reserved chars. Solving this for Windows should be quite easy though and just splitting on the first `:`:

```
licensecheck --using requirements:c:/path/to/reqs.txt;path/to/other/reqs.txt
```

**Comment by Niccolum** [comment]
What do you think about next:
At first, I don't know any person, who use requirements.txt and poetry in one project, so, we can still use `--using` flag, as you already created.
`licensecheck --using requirements`
but for path to file, I think, be better to use another flag, for example `--path`. And, as I know, we can use `nargs=+` or `append` option. Here is examples
https://stackoverflow.com/a/15753721
My main point is not to mix the purpose of two different flags - the type of requirements and the path to them.

**Comment by FredHappyface** [comment]
fixed in 2021.4
